modules/icon/react/lib/Svg.tsx

```diff
@@ -40,8 +35,7 @@ export default class Svg extends React.Component<SvgProps> {
               css(styles, {
                 display: 'inline-block',
                 '& svg': {display: 'block'},
-              }),
-              elemProps.className
```

Comment:
because we restructured this, the test that was written to test this fails. Do we still want this functionality of overwriting the class name with a new prop? @anicholls

Reply:
@mannycarrera4 It was correct remove the `defaultProps` on line 16, but this didn't need to go anywhere. We should keep this functionality. You can add `elemProps.className` back in here and the test should pass